tests/test_types.py

```diff
@@ -0,0 +1,135 @@
+r"""*Submodule for test suite of the* ``stdio_mgr.types`` components.
+
+**Author**
+    John Vandenberg (jayvdb@gmail.com)
+
+**File Created**
+    6 Sep 2019
+
+**Copyright**
+    \(c) Brian Skinn 2018-2019
+
+**Source Repository**
+    http://www.github.com/bskinn/stdio-mgr
+
+**Documentation**
+    See README.rst at the GitHub repository
+
+**License**
+    The MIT License; see |license_txt|_ for full license terms
+
+**Members**
+
+"""
+
+import io
+
+import pytest
+
+from stdio_mgr.stdio_mgr import _IMPORT_SYS_STREAMS, _RUNTIME_SYS_STREAMS
+from stdio_mgr.types import ClosingStdioTuple, FakeIOTuple, TextIOTuple
+
+
+def _create_dummy_wrapper():
```

Comment:
This and `..._tuple()`, below, might be good to refactor into root `conftest.py` as fixtures, returning a function object.

Could see them being useful across the suite.